From 2b444ccd423e089abb2cc4c44f9c85fae653aacb Mon Sep 17 00:00:00 2001
From: Jason Chen <b02280@freescale.com>
Date: Wed, 23 Jun 2010 14:28:15 +0800
Subject: [PATCH 160/505] ENGR00124360 v4l2 output: change unit test

IC bypass mode has limitation for output count, it only can output
(dqueue count -1) frames.

Signed-off-by: Jason Chen <b02280@freescale.com>
---
 test/mxc_v4l2_test/mxc_v4l2_output.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/test/mxc_v4l2_test/mxc_v4l2_output.c b/test/mxc_v4l2_test/mxc_v4l2_output.c
index 03e7bff..f8ecd76 100644
--- a/test/mxc_v4l2_test/mxc_v4l2_output.c
+++ b/test/mxc_v4l2_test/mxc_v4l2_output.c
@@ -241,7 +241,7 @@ mxc_v4l_output_test(FILE *in)
 						if (i < g_num_buffers)
 							deq_cnt = i;
 						else
-							deq_cnt = g_num_buffers;
+							deq_cnt = g_num_buffers - 1;
 						for (j=0;j<(deq_cnt - 1);j++) {
 							buf.type = V4L2_BUF_TYPE_VIDEO_OUTPUT;
 							buf.memory = V4L2_MEMORY_MMAP;
@@ -273,7 +273,7 @@ mxc_v4l_output_test(FILE *in)
 						if (i < g_num_buffers)
 							deq_cnt = i;
 						else
-							deq_cnt = g_num_buffers;
+							deq_cnt = g_num_buffers - 1;
 						for (j=0;j<(deq_cnt - 1);j++) {
 							buf.type = V4L2_BUF_TYPE_VIDEO_OUTPUT;
 							buf.memory = V4L2_MEMORY_MMAP;
-- 
1.8.0

